docs/reference/upgrade/rolling_upgrade.asciidoc

```diff
@@ -164,7 +164,20 @@ recovery completes.
 --
 
 When  the node has recovered and the cluster is stable, repeat these steps
-for each node that needs to be updated.
+for each node that needs to be updated. You can monitor the health of the cluster
+with a <<cat-health,`_cat/health`>> request:
+
+[source,console]
+--------------------------------------------------
+GET /_cat/health?v
+--------------------------------------------------
+
+And the version of each node with a <<cat-nodes,`_cat/nodes`>> request:
```

Comment:
Maybe word it like: `And see which nodes have been upgraded with a  <<cat-nodes,`_cat/nodes`>> request:` ?